org.eclipse.xtext.builder/.classpath

```diff
@@ -2,13 +2,9 @@
 <classpath>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="src" path="xtend-gen"/>
-	<classpathentry kind="src" path="emf-gen">
-		<attributes>
-			<attribute name="ignore_optional_problems" value="true"/>
-		</attributes>
-	</classpathentry>
-	<classpathentry kind="src" path="deprecated"/>
-	<classpathentry kind="output" path="bin"/>
+	<classpathentry kind="src" path="emf-gen/"/>
+	<classpathentry kind="src" path="src/"/>
```

Comment:
why emf-gen before src?

Reply:
Seems to depend on the order in the build.properties (which does make sense). I'll reorder it there.

Reply:
FYI The order can be defined in the build path and affects the order in which the containers are listed in the navigator tree.

Reply:
See next commit